src/Microsoft.ML.Transforms/CountFeatureSelection.cs

```diff
@@ -31,22 +31,23 @@ namespace Microsoft.ML.Transforms
     /// | Input column data type | Vector or scalar of numeric, [text](xref:Microsoft.ML.Data.TextDataViewType) or [key](xref:Microsoft.ML.Data.KeyDataViewType) data types|
     /// | Output column data type | Same as the input column|
     ///
-    /// This transform uses a set of aggregators to count the number of non-default values for each slot(vector element) and
-    /// instantiates a transformer to actually drop the slots that have the default values for the types
-    /// keeping at least 'count' non-default values.
+    /// This transform uses a set of aggregators to count the number of values for each slot (vector element)
+    /// that are non-default and non-missing (for the definitions of default and missing, refer to the remarks section
+    /// in <xref:Microsoft.ML.Data.DataKind>). If the count value is less than the provided count parameter, that slot is dropped.
     /// This transform is useful when applied together with a <xref:Microsoft.ML.Transforms.OneHotHashEncodingTransformer>.
-    /// The count feature selection can remove the features generated by the hash transform that have no data in the examples.
+    /// It can remove the features generated by the hash transform that have no data in the examples.
```

Comment:
Could you add a reference to hash transfomer's API?

Reply:
there's an xref link to OneHotHashEncodingTransformer in the line above. Is that what you mean?

---
In reply to: [290927868](https://github.com/dotnet/machinelearning/pull/3831#discussion_r290927868) [](ancestors = 290927868)

Reply:
I meant
```c#
            var pipeline = mlContext.Transforms.Categorical.OneHotHashEncoding(
                "EducationOneHotHashEncoded", "Education", numberOfBits: 3);
```